src/vimodes/vimodefpalhaf1.c

```diff
@@ -0,0 +1,35 @@
+#include "PR/os.h"
+#include "PR/rcp.h"
+#include "../io/viint.h"
+
+OSViMode osViModeFpalHaf1 = {
+    OS_VI_FPAL_HAF1,  // type
+    { // comRegs
+        VI_CTRL_TYPE_16 | VI_CTRL_GAMMA_DITHER_ON | VI_CTRL_GAMMA_ON |
+            VI_CTRL_DIVOT_ON | VI_CTRL_SERRATE_ON | 0x3000,  // ctrl
```

Comment:
```suggestion
            VI_CTRL_DIVOT_ON | VI_CTRL_SERRATE_ON | VI_CTRL_PIXEL_ADV_3,  // ctrl
```
(also in other individual vimode files)
This constant isn't actually standard in libultra headers, I read about it being related to "Pixel Advance" in https://dragonminded.com/n64dev/Reality%20Coprocessor.pdf
I think it's preferable over a constant whatever the case